### PR TITLE
Add outline for LSFF concept model

### PR DESCRIPTION
**Pull request by NathanielBlairStahn**
Starting documentation for Large Scale Food Fortification concept model

### CODE REVIEW

**Comment by NathanielBlairStahn** [comment]
@aflaxman 
> DId @collijk convince you to merge all three fortification concept models into one document?

I haven't figured out the best way to do it yet.  @collijk didn't give me any specific suggestions. Right now I have three separate documents (one for each nutrient) so I can focus in on as much detail as necessary, plus a master LSFF document where I can talk about them together. I'll have to work out the best way to organize everything as I go; I'm open to suggestions.

**Comment by collijk** [comment]
@NathanielBlairStahn and I just talked this through.  He's restructuring PRs shortly

**Comment by NathanielBlairStahn** [comment]
@collijk I made a single concept model file like we discussed. What do you think of the sections and organization in the outline?

File: docs/source/models/gbd2017/concept_models/vivarium_conic_lsff/concept_model.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+Objective
++++++++++
+
+Intervention Definitions
+++++++++++++++++++++++++
+
+Questions of Interest
++++++++++++++++++++++
+
+Scope of Modeling
```
Comment (collijk) [inline docs/source/models/gbd2017/concept_models/vivarium_conic_lsff/concept_model.rst]: Nice translation

**Comment by NathanielBlairStahn** [comment]
> We'll probably need to shift some of these things around, but looks good so far!

Cool, thanks. Yeah, it's hard to know how to organize it before the content is there.

**Comment by yongqx** [comment]
Some items came to my mind for concept model note are:
1. Data sources (state measure + transition)
2. We should separate GBD cause & risk into two pieces 
3. Clarify coverage definition, as well as intervention definition

- food fortification level vs. proportion of population eating fortified food
- fortification vs. supplementation

**Comment by NathanielBlairStahn** [comment]
> Some items came to my mind for concept model note are:
> 
> 1. Data sources (state measure + transition)
> 2. We should separate GBD cause & risk into two pieces
> 3. Clarify coverage definition, as well as intervention definition
> 
> * food fortification level vs. proportion of population eating fortified food
> * fortification vs. supplementation

Thanks for the suggestions, @yongqx . I'll be adding in more detail as I fill in the sections. For the data sources, I was thinking most of that information would go in the cause model documentation, not in the concept model. Are there specific pieces of data that you think belong in the concept model itself (e.g. if they aren't tied to a particular cause or risk)?

**Pull request merged**